src/APIM_ARMTemplate/apimtemplate/Common/Logger.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+
+namespace apimtemplate.Common
+{
+    /// <summary>
+    /// Helper class for logging any actions. 
+    /// You can setup logger passing Microsoft.Extensions.Logging.ILogger that works the best for you.
+    /// </summary>
+    public static class Logger
```

Comment:
What is the added value for this wrapper? We seem to just pass the calls to the underlying logger